torch/_dynamo/symbolic_convert.py

```diff
@@ -2498,7 +2498,11 @@ def inline_call_(
             args
             and isinstance(
                 args[0],
-                (variables.CustomizedDictVariable, variables.UserDefinedObjectVariable),
+                (
+                    variables.CustomizedDictVariable,
+                    variables.UserDefinedObjectVariable,
+                    variables.NNModuleVariable,
```

Comment:
What does this do? It's hard to wrap my head around the condition, but is it that you can now define `__setattr__` on a `nn.module`? 

If so does it get inlined? 

Why was it banned before?

Reply:
Ah sorry, I think this was part of another direction I was trying to take to fix the const attr issue -- I deleted this change and looks like the tests still work 😅